src/components/navbarClient.tsx

```diff
@@ -14,10 +15,12 @@ interface Props {
 }
 
 export function NavbarClient({platforms, currentPlatform}: Props) {
+  const pathname = usePathname() || undefined;
```

Comment:
tiny nit: does `??` make more since. It's just a bit more strict.

Reply:
swapped, thanks!